eZ/Bundle/EzPublishCoreBundle/Tests/DependencyInjection/EzPublishCoreExtensionTest.php

```diff
@@ -889,9 +889,11 @@ public function testQueryTypeAutomaticConfiguration(): void
      */
     private function compileCoreContainer(): void
     {
+        $this->disableCheckExceptionOnInvalidReferenceBehaviorPass();
```

Comment:
In this particular case, I decided to disable `CheckExceptionOnInvalidReferenceBehaviorPass` as it throws `ServiceNotFoundException` on the high number of services that are not the subject of the test e.g. `event_dispatcher`, `http_kernel`.